CHANGELOG.md

```diff
@@ -61,6 +61,8 @@ decommissioned.
   flag to `forest-tool api compare`.
 - [#3806](https://github.com/ChainSafe/forest/pull/3806) Implement the
   `Filecoin.EthGasPrice` lotus-compatible RPC API.
+- [#3866](https://github.com/ChainSafe/forest/pull/3866) Implement Offline RPC
```

Comment:
This should be moved to `unreleased` section

Reply:
i'll fix it

Reply:
fixed